snippets/common/proxy-params.conf

```
@@ -5,7 +5,11 @@ proxy_set_header Upgrade            $http_upgrade;
 proxy_set_header Connection         "upgrade";
 
 proxy_set_header X-Real-IP          $remote_addr;
-proxy_set_header X-Forwarded-For    $proxy_add_x_forwarded_for;
+
+# We do not need to do anything here, because CloudFlare will set the X-Forwarded-For header for us.
+# but this needs to be changed if you are using a different proxy.
+
+# proxy_set_header X-Forwarded-For    $proxy_add_x_forwarded_for;
```

Sorry to keep coming back to this, but CF trusts the XFF header it gets and that means we can't use it as a key.  `CF-Connecting-IP` is fine, as is `X-Real-IP` (i.e. what $remote_addr gets set to), because the [realip module](https://nginx.org/en/docs/http/ngx_http_realip_module.html) puts `CF-Connecting-IP`'s into `$remote_addr`.

I think the best plan is to do this:
```suggestion
proxy_set_header X-Forwarded-For    $remote_addr;
```

i.e. rely on the realip module.